com.unity.ml-agents.extensions/Runtime/Match3/Match3Actuator.cs

```diff
@@ -5,16 +5,31 @@
 
 namespace Unity.MLAgents.Extensions.Match3
 {
+    public enum HeuristicQuality
```

Comment:
I don't think we need this in the base class, since by default it will have no effect - it switches between hardcoded 1.0 and virtual method that returns 1.0.

Reply:
(and just use `var movePoints = EvalMovePoints(move);` below)

Reply:
good call